turbinia/workers/__init__.py

```diff
@@ -563,16 +566,19 @@ def execute(
       rw_paths = [self.output_dir, self.tmp_dir]
       container_manager = docker_manager.ContainerManager(docker_image)
       stdout, stderr, ret = container_manager.execute_container(
-          cmd, shell, ro_paths=ro_paths, rw_paths=rw_paths)
+          cmd, shell, ro_paths=ro_paths, rw_paths=rw_paths,
```

Comment:
Since this is getting implemented as part of the `execute()` method, can you check the other Tasks to see which of them are executing without using this helper method?  Maybe we can convert them to use `execute()`.